test/unit/model/storage/SqlTest.php

```diff
@@ -56,22 +53,22 @@ public function testCastValue($fieldName, $value, $expectedType)
     public function valueProvider()
     {
         return [
-            ['size', 1, 'integer'],
-            ['size', 't1', 'string'],
-            [Sql::DIAGNOSTIC_BANDWIDTH_SIZE, '12', 'integer'],
-            [Sql::DIAGNOSTIC_INTENSIVE_BANDWIDTH_SIZE, '12.00', 'integer'],
-            [Sql::DIAGNOSTIC_PERFORMANCE_COUNT, '12.33', 'integer'],
-            [Sql::DIAGNOSTIC_BANDWIDTH_COUNT, 22, 'integer'],
-            [Sql::DIAGNOSTIC_INTENSIVE_BANDWIDTH_COUNT, '33.02', 'integer'],
-            [Sql::DIAGNOSTIC_OSVERSION, 10.0, 'string'],
-            [Sql::DIAGNOSTIC_OSVERSION, '10.0', 'string'],
-            [Sql::DIAGNOSTIC_OSVERSION, 8, 'string'],
-            [Sql::DIAGNOSTIC_OSVERSION, '8', 'string'],
-            [Sql::DIAGNOSTIC_BROWSERVERSION, 33.02, 'string'],
-            [Sql::DIAGNOSTIC_BROWSERVERSION, 73, 'string'],
-            ['another', '33.02', 'float'],
-            ['another3', 'omg', 'string'],
-            ['another2', 33.02, 'float'],
+            ['size', 1, 'assertIsInt'],
+            ['size', 't1', 'assertIsString'],
+            [Sql::DIAGNOSTIC_BANDWIDTH_SIZE, '12', 'assertIsInt'],
+            [Sql::DIAGNOSTIC_INTENSIVE_BANDWIDTH_SIZE, '12.00', 'assertIsInt'],
```

Comment:
this is a little bit weird

Reply:
Yes. i'm also don't like this solution. But i have no other ideas to quick fix this.

Reply:
I agree. It's not but this fix keep consistent with other changes so I'm willing to accept it.